dds_web/database/models.py

```diff
@@ -115,7 +49,8 @@ class Project(db.Model):
     __table_args__ = {"extend_existing": True}
 
     # Columns
-    id = db.Column(db.String(32), primary_key=True)
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    public_id = db.Column(db.String(32), unique=True, nullable=False)
```

Comment:
Just occurred, since we will using this field will mostly used elsewhere (for queries and etc) could we name this column as `id` and change the primary key column to `db_id` or `secret_id` or something

Reply:
I think it's like a standard to have `id` as the primary key column? We don't have to, but personally it feels like this could cause even more confusion, but we can discuss it if you want.

Reply:
Yes, just thinking as DB entity `id` is more suitable for a primary key column. But while working for frontend, during display or to fetch some info using `id` (real project id which is displayed) I first instinctively write `id` then will change to `public_id`. That is why I thought could just have `id` for `id`, maybe `key` or something else for primary key